cache/fast_lru_cache.h

```diff
@@ -304,7 +303,7 @@ class LRUCache
 }  // namespace fast_lru_cache
 
 std::shared_ptr<Cache> NewFastLRUCache(
-    size_t capacity, int num_shard_bits = -1,
```

Comment:
I believe there are 3 more callers to NewFastLRUCache that are passing in a num_shard_bits as the second parameter, which is now being used as `estimated_value_size`. Eventually we will want to move toward an Options object like LRUCacheOptions to avoid such bugs, but that might slow down prototyping for now. But while this is internal-only, it is probably best to remove optional parameters to avoid this kind of bug.

Reply:
My bad. I should've done a grep. Thanks!